tests/php/Security/MemberAuthenticatorTest.php

```diff
@@ -109,140 +60,137 @@ public function testCustomIdentifierField()
 
     public function testGenerateLoginForm()
     {
+        $authenticator = new Authenticator();
+
         $controller = new Security();
 
         // Create basic login form
-        $frontendForm = MemberAuthenticator::get_login_form($controller);
-        $this->assertTrue($frontendForm instanceof MemberLoginForm);
+        $frontendResponse = $authenticator
+            ->getLoginHandler($controller->link())
+            ->handleRequest(new HTTPRequest('get', '/'), \SilverStripe\ORM\DataModel::inst());
+
+        $this->assertTrue(is_array($frontendResponse));
+        $this->assertTrue(isset($frontendResponse['Form']));
+        $this->assertTrue($frontendResponse['Form'] instanceof LoginForm);
+    }
+
+    /* TO DO - reenable
+    public function testGenerateCMSLoginForm()
```

Comment:
What is it exactly supposed to do? The CMSLoginForm is horrible currently, but the test itself is a bit unclear on what it is trying to achieve @dhensby @sminnee @tractorcow @chillu